commentHandler.go

```diff
@@ -12,8 +13,17 @@ import (
 	"github.com/google/go-github/github"
 )
 
-const open = "open"
-const closed = "closed"
+const openConstant string = "open"
+const closedConstant string = "closed"
+const closeConstant string = "close"
+const reopenConstant string = "reopen"
+const lockConstant string = "Lock"
+const unlockConstant string = "Unlock"
```

Comment:
Is there a mixture of command words and actions to take? like `close` and `Lock`?